### ISSUE TITLE
[Addon] Add Let's Encrypt ACME Configuration in `traefik` Addon

### ISSUE THREAD

**Issue by charlie0129** [kubevela/catalog#368]
The `traefik` helm chart has included support for ACME certificate resolver configuration through helm Values: https://github.com/traefik/traefik-helm-chart/pull/419

Currently, we need to manually input `cert` and `key` with `traefik` addon.

It is better if Let's Encrypt ACME is supported.

- [ ] Support multiple domains in cert-manager.
- [ ] Tutorial on how to use cert-manager and traefik together

**Comment by barnettZQG** [comment]
@charlie0129 Good proposal, do you contribute?

**Comment by charlie0129** [comment]
Sure, I will work on this later.

**Comment by wonderflow** [comment]
why don't add let's encriypt as an addon, so other addon can dependsOn it if they need mTLS.

**Comment by charlie0129** [comment]
IMO, it is just used for one-way TLS on the public Internet (or k8s Ingress)? Not so sure if it is widely used for mTLS.

**Comment by wonderflow** [comment]
Yes, it's popular, you can check: https://cloud.redhat.com/blog/using-kubernetes-operators-to-manage-lets-encrypt-ssl-tls-certificates-for-red-hat-openshift-dedicated

I'm not sure if there's any better alternatives.

**Comment by charlie0129** [comment]
Very informative. I will check how it is achievable using addons.
